packages/elements-dev-portal/src/handlers/getBranches.ts

```diff
@@ -11,7 +12,7 @@ export const getBranches = async ({
 }): Promise<Branch[]> => {
   const response = await fetch(`${platformUrl}/api/v1/projects/${projectId}/branches`, {
     headers: {
-      'Stoplight-Elements-Version': '1.0.0',
+      'Stoplight-Elements-Version': appVersion,
```

Comment:
At this point I am thinking we want some kind of wrapper for fetch, which will encapsulate this and getting a token for the request. 🤷

Reply:
middleware! middleware! 

Maybe something like https://github.com/benjamine/fetch-wrap or any other middleware implementation that looks good. This will let us add all sorts of cool generic functionality, like generic http caching, sunset sniffing, auto-retries, circuit breakers, etc.

Reply:
We could use either some middleware package or just wrap it in a simple module, but we would still need to pass the auth token along url.